user-guide/Cloud_Platform/Catalog/Deploying_A_DataMiner_Connector_to_your_system.md

```diff
@@ -21,7 +21,9 @@ To deploy an item from the DataMiner Catalog (e.g. a connector or package) to yo
    > [!NOTE]
    >
    > - The *Deploy* button is only available if your organization has a license for the displayed item. If it is unavailable, to be able to deploy the item, contact <licensing@skyline.be>.
```

Comment:
maybe here add an xref to the organization verification, only when this is done we can check the linked license